File: boj/prob01076.py
# Backjoon Online Judge: https://www.acmicpc.net/problem/1076
# 내용없음

import sys

res = []
for i in range(3):
    res.append(sys.stdin.readline().rstrip())
colors = ['black', 'brown', 'red', 'orange', 'yellow', 'green', 'blue', 'violet', 'grey', 'white']
resistance = {color: (i, 10 ** (i)) for i, color in enumerate(colors)}
print((resistance[res[0]][0] * 10 + resistance[res[1]][0]) * resistance[res[2]][1])

File: boj/prob01076.java
package boj;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.HashMap;
import java.util.Map;

class prob01076 {
    static final Map<String, Integer> value = new HashMap<String, Integer>();
    static final Map<String, Integer> multiple = new HashMap<String, Integer>();

    public static void main(String[] args) throws IOException {
        init();
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        long result = 0;
        result += 10 * value.get(br.readLine());
        result += value.get(br.readLine());
        result *= multiple.get(br.readLine());
        System.out.println(result);
    }

    public static void init() {
        String[] colors = { "black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white" };
        for (int i = 0; i < colors.length; i++) {
            value.put(colors[i], i);
            multiple.put(colors[i], (int) Math.pow(10, i));
        }
    }
}